CHANGELOG.md

```diff
@@ -1,66 +1,106 @@
 # Changelog
+
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.116.0] - 2023-03-28
+
+### Added
+
+- Smart Transfer Protocol: implemented Pull flow based on web-sockets
+- S3-based dataset repository implementation
+
+### Changed
+
+- Improved status codes and error handling in API server's routes for Simple Transfer Protocol
+
 ## [0.115.0] - 2023-03-19
+
```

Comment:
I prefer the compact format (and changed changelog formats accordingly in few other projects), so if you're not proposing a new style - let's revert these whitespace changes.

Reply:
Restored